modules/simphera_instance/postgresql.tf

```diff
@@ -4,22 +4,34 @@ resource "azurerm_resource_group" "postgres" {
   tags     = var.tags
 }
 
+resource "random_password" "postgresql-rnd-pass" {
```

Comment:
postgresql-rnd-pass => postgresql-password
(As a matter of taste, I prefer avoiding abbreviations)